pipeline_products_session/jwst-data-products-part1.ipynb

```diff
@@ -0,0 +1,961 @@
+   "outputs": [],
+   "source": [
+    "fits.info(uncal_obs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can grab the data to examine it: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data = fits.getdata(uncal_obs,1)\n",
+    "\n",
+    "# or \n",
+    "\n",
+    "with fits.open(uncal_obs) as hdu:\n",
+    "    science_data = hdu['SCI'].data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The science data shape here shows the number of integrations, groups, rows (pixels), and columns (pixels), which reflects the up-the-ramp readout (also referred to as MULTIACCUM) standardized readout sampling for all JWST detectors (read more in the [JWST User Documentation](https://jwst-docs.stsci.edu/understanding-exposure-times)). We'll talk about this more in the following sections. For now, let's look at the associated headers and other metadata.                                                    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### B.<font color='white'>-</font>Metadata<a class=\"anchor\" id=\"astro-meta\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Headers containing information about the observation and the data parameters be accessed the standard way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "primary_headers = fits.getheader(uncal_obs,0)\n",
+    "science_headers = fits.getheader(uncal_obs,1)\n",
+    "\n",
+    "# or \n",
+    "\n",
+    "with fits.open(uncal_obs) as hdu:\n",
+    "    primary_headers = hdu['PRIMARY'].header\n",
+    "    science_headers = hdu['SCI'].header"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Observation ID: ', primary_headers['OBS_ID'])\n",
+    "print('Instrument: ', primary_headers['INSTRUME'])\n",
+    "print('Exposure type: ', primary_headers['EXP_TYPE'])\n",
+    "print('Detector: ', primary_headers['DETECTOR'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('\\nNumber of data dimensions: ', len(science_data.shape))\n",
+    "print('Number of integrations: ', primary_headers['NINTS'])\n",
+    "print('Number of groups: ', primary_headers['NGROUPS'])\n",
+    "print('Number of rows: ', primary_headers['SUBSIZE1'])\n",
+    "print('Number of columns: ', primary_headers['SUBSIZE2'])\n",
+    "\n",
+    "# or \n",
+    "\n",
+    "print('\\nNumber of data dimensions: ', science_headers['NAXIS'])\n",
```

Comment:
If the goal is to start to introduce users to JWST jargon, maybe we should remove the NAXIS lines here and keep only the lines that use the primary header keywords?

Reply:
Good point, removed NAXIS reference